lib/rules/media-query-no-invalid/README.md

```diff
@@ -62,3 +62,39 @@ The following patterns are _not_ considered problems:
 ```css
 @media (color) and ((hover) or (width)) {}
 ```
+
+## Optional secondary options
+
+### `ignoreFunctions: ["/regex/", /regex/, "string"]`
+
+Ignore the specified functions.
+
+For example, with `true`.
+
+Given:
+
+```json
+["theme", "/^get.*$/"]
+```
+
+The following patterns are _not_ considered problems:
+
+<!-- prettier-ignore -->
+```css
+@media (min-width: theme(screens.md)) {}
+```
+
+<!-- prettier-ignore -->
+```css
+@media (min-width: getDefaultWidth()) {}
+```
+
+<!-- prettier-ignore -->
+```css
+@media (max-width: get-default-width()) {}
+```
+
+<!-- prettier-ignore -->
+```css
+@media (max-width: get_default_width()) {}
+```
```

Comment:
[suggestion] The 2 examples for `"/^get.*$/"` seem redundant. I think having only one example is enough.

Personally, `get-default-width` is preferred, but it's not a strong opinion.

Reply:
Thank you for all the comments!
I've applied your suggestion.
[apply code review suggestion](https://github.com/stylelint/stylelint/pull/8060/commits/de4e39246897a8378945352306b0538f8afd0515)